src/components/SubmitResponsePage.jsx

```diff
@@ -56,37 +55,22 @@ const SubmitResponse = (props) => {
   };
   const HomeButton = (
     <React.Fragment>
-      <ButtonDisplay
-        key={1}
-        onClick={returnHome}
-        buttonName="Create New Report"
-        cssClass="seButton"
-      />
+      <SeButton type="button" onClick={returnHome} text="Create New Report" />
```

Comment:
There wasnt a need to have this ButtonDisplay component any longer since our library button component handles its custom changes